modules/plugin/src/main/scala/org/scalasteward/plugin/StewardPlugin.scala

```diff
@@ -96,6 +102,13 @@ object StewardPlugin extends AutoPlugin {
   private def isCompilerPlugin(moduleId: ModuleID): Boolean =
     moduleId.configurations.exists(_ == "plugin->default(compile)")
 
+  lazy val findScalafixDependencies: Def.Initialize[Seq[ModuleID]] = Def.settingDyn {
+    val scalafixDependencies = SettingKey[Seq[ModuleID]]("scalafixDependencies")
+    Def.setting {
+      scalafixDependencies.value
```

Comment:
That looks interesting. What happens if a project does not depend on sbt-scalafix and this is evaluated? Will it just return an empty `Seq` or does it throw?

Reply:
I realized that I had `sbt-scalafix` in my global plugins 😞 
So I went back to the code from `mdoc` I had used as an inspiration and fixed my implementation.
I tested it with a  test project with and  with another without `sbt-scalafix`

Reply:
It is cool that this works!